src/backend/distributed/operations/shard_rebalancer.c

```diff
@@ -1177,13 +1501,9 @@ UpdateShardPlacement(PlacementUpdateEvent *placementUpdateEvent,
 												   targetNode->workerPort);
 	if (!targetResponsive)
 	{
-		ereport(WARNING, (errmsg("%s:%d is not responsive", targetNode->workerName,
-								 targetNode->workerPort)));
-		UpdateColocatedShardPlacementProgress(shardId,
```

Comment:
seems to be the only place from where we call this function, don't see this function removed from our source with this PR.

Would there be a reason we keep this function around?

Reply:
It's still used a few lines below.

Reply:
ah, thanks, only searched for functions it was called from, not the occurrences within that function :)